Sources/ConsoleKit/Command/CommandError.swift

```diff
@@ -5,15 +5,34 @@ public struct CommandError: Error, CustomStringConvertible {
 
     /// See `Debuggable`.
     public let reason: String
+    
+    let name: String?
+    let commands: [String]
 
     /// See `CustomStringConvertible`.
     public var description: String {
-        return "\(self.identifier): \(self.reason)"
+        guard let name = name else {
+            return "\(self.identifier): \(self.reason)"
+        }
+        
+        return """
+        Error: \(self.reason)
+
+        Did you mean this?
+        
+            \(suggestions(for: name, in: self.commands).joined(separator: "\n\t"))
```

Comment:
If the resulting array from calling `suggestions(for:in:)` is empty, we don't want the suggested command section in the error message. Instead we should probably get the suggested commands, and then if the array has 1 or more elements, then output this error message. Otherwise we can just output `Command with name '<name>' not found`.

Reply:
Absolutely, as noted in the PR